sysutils/scp-backup/src/opnsense/mvc/app/models/OPNsense/ScpBackup/General.xml

```diff
@@ -33,11 +33,20 @@
           <Required>Y</Required>
           <ValidationMessage>Please provide a remote directory.</ValidationMessage>
         </remotedirectory>
-        <publickey type="TextField">
-          <Required>N</Required>
-        </publickey>
+        <privatekey type="TextField">
```

Comment:
should be a ModelRelationField
Field: key_pair
Model: OPNsense\Sshkeys\SSH

and display key_name